staging/src/k8s.io/apimachinery/pkg/util/net/http.go

```diff
@@ -132,7 +133,7 @@ func SetTransportDefaults(t *http.Transport) *http.Transport {
 	if s := os.Getenv("DISABLE_HTTP2"); len(s) > 0 {
 		klog.Infof("HTTP2 has been explicitly disabled")
 	} else if allowsHTTP2(t) {
-		if err := http2.ConfigureTransport(t); err != nil {
+		if err := http2.ConfigureTransport(t, http2.WithReadIdleTimeout(3*time.Second), http2.WithPingTimeout(2*time.Second)); err != nil {
```

Comment:
This default value is too frequent, i think 30s is appropriate

Reply:
I have updated ReadIdleTimeout(30s) and PingTimeout(1s)

Reply:
30s might be too infrequent, but we can tune it later.